source/common/quic/envoy_quic_utils.cc

```diff
@@ -14,6 +14,18 @@
 namespace Envoy {
 namespace Quic {
 
+namespace {
+
+Network::Address::InstanceConstSharedPtr
+getLoopbackAddress(const Network::Address::IpVersion version) {
+  if (version == Network::Address::IpVersion::v6) {
+    return std::make_shared<Network::Address::Ipv6Instance>("::1");
+  }
+  return std::make_shared<Network::Address::Ipv4Instance>("127.0.0.1");
```

Comment:
There are existing util functions for loopback address, i.e. `Network::Utility::getCanonicalIpv4LoopbackAddress()`.